example/90-shoot.yaml

```diff
@@ -67,7 +67,7 @@ spec:
     #   effect: NoSchedule
     # caBundle: <some-ca-bundle-to-be-installed-to-all-nodes-in-this-pool>
     # kubernetes:
-    #   kubelet:
+    #   kubelet: # if configured all `spec.kubernetes.kubelet` settings will be overwritten for this worker group (no merge of settings).
```

Comment:
I believe the most appropriate place for such details is the doc string of the corresponding field -> it will then part of the API reference - . We don't explain each and every field in the example files.

Reply:
Probably both would be fine.